app/views/payments/OpenPayments.scala.html

```diff
@@ -77,19 +79,26 @@ <h1 class="govuk-heading-xl">@messages("openPayments.heading")</h1>
           <p class="govuk-body">@messages("openPayments.processingTime")</p>
         </div>
 
-        <h2 class="govuk-heading-s" id="what-you-owe-missing">@messages("openPayments.whatOweHelp.heading")</h2>
-        <p class="govuk-body">@messages("openPayments.incorrect")
+        <h2 class="govuk-heading-s" id="what-you-owe-missing">
+            @if(user.isAgent) {@messages("openPayments.whatOweHelp.headingAgent")} else {@messages("openPayments.whatOweHelp.heading")}
+        </h2>
+
+        <p class="govuk-body">
+            @if(user.isAgent) {@messages("openPayments.incorrectAgent")} else {@messages("openPayments.incorrect")}
+
           <a class="govuk-link" href="https://www.gov.uk/vat-corrections" rel="noreferrer noopener" target="_blank">
-            @messages("openPayments.correctErrors")</a>.
+              @if(user.isAgent) {@messages("openPayments.correctErrorsAgent")} else {@messages("openPayments.correctErrors")}</a>@messages("common.fullStop")
+
         </p>
         <p class="govuk-body">
-          @messages("payment.afterSubmitted")
-          @if(!user.isAgent) {
-            @messages("payment.stillMake")
-            <a class="govuk-link" href="@appConfig.unauthenticatedPaymentsUrl"
-               id="vatPaymentsLink" rel="noreferrer noopener" target="_blank">
-              @messages("payment.stillMakeLink")
-              <span class="govuk-visually-hidden">@messages("payment.evenIfNotShown")</span></a>.
+            @if(user.isAgent) {@messages("noPayments.twentyFourHoursAgent")} else {@messages("payment.afterSubmitted")}
+
+              @if(!user.isAgent) {
+                @messages("payment.stillMake")
+                <a class="govuk-link" href="@appConfig.unauthenticatedPaymentsUrl"
+                   id="vatPaymentsLink" rel="noreferrer noopener" target="_blank">
+                  @messages("payment.stillMakeLink")
+                  <span class="govuk-visually-hidden">@messages("payment.evenIfNotShown")</span></a>@messages("common.fullStop")
```

Comment:
Think this line still needs to go up to correct the issue on client side. Not sure why this one in particular is not caught in unit tests - perhaps down to the function that extracts text from element. Once again - this part is not a result of your change but while we're here, etc.

![image](https://user-images.githubusercontent.com/29063729/146959499-45a5b685-b1c0-4432-b319-fcc90719219d.png)